backend/app/services/users/sign_out.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Users
+  class SignOut < Core::Service
+    def call(headers)
```

Comment:
Let pass `current_user` here directly instead of `headers` and make this named argument with `user:`